open-metadata-implementation/access-services/asset-catalog/asset-catalog-server/src/main/java/org/odpi/openmetadata/accessservices/assetcatalog/handlers/CommonHandler.java

```diff
@@ -137,7 +137,7 @@ public EntityDetail getEntityByGUID(String userId,
         try {
             return repositoryHandler.getMetadataCollection().getEntityDetail(userId, guid);
         } catch (org.odpi.openmetadata.repositoryservices.ffdc.exception.InvalidParameterException e) {
-            e.printStackTrace();
+            errorHandler.handleRepositoryError(e, methodName);
```

Comment:
I suggest we check to see if the invalid parameter is the guid and then call errorHandler.handleUnknownEntity - so we do not lose this insight.